packages/core/src/components/table/table-body/table-body.scss

```diff
@@ -19,3 +19,75 @@ tds-table-body {
     transition: background-color 200ms ease;
   }
 }
+
+// Zebra mode: rows
```

Comment:
Nice! If we want to we could DRY it up some more:

```
// Zebra mode: rows (odd and even)

// Combine odd and even row selectors
tds-table-body.tds-table--zebra-mode-rows-odd tds-table-body-row:nth-child(odd),
tds-table-body.tds-table--zebra-mode-rows-even tds-table-body-row:nth-child(even) {
  background-color: var(--tds-table-zebra-mode-backround);
}

tds-table-body.tds-table--zebra-mode-rows-odd tds-table-body-row:nth-child(odd):hover,
tds-table-body.tds-table--zebra-mode-rows-even tds-table-body-row:nth-child(even):hover {
  background-color: var(--tds-table-body-row-background-hover);
}

// Zebra mode: columns (odd and even)

// Combine odd and even column selectors
tds-table-body.tds-table--zebra-mode-columns-odd tds-body-cell:nth-child(odd),
tds-table-body.tds-table--zebra-mode-columns-even tds-body-cell:nth-child(even) {
  background-color: var(--tds-table-zebra-mode-backround);
}

tds-table-body.tds-table--zebra-mode-columns-odd tds-body-cell.tds-table__body-cell--hover:nth-child(odd),
tds-table-body.tds-table--zebra-mode-columns-even tds-body-cell.tds-table__body-cell--hover:nth-child(even) {
  background-color: var(--tds-table-body-cell-background-hover);
}

```
Just combining some of the repeating parts. But it comes down to personal preference :)

Reply:
Done!